src/dev/ci_setup/setup.sh

```diff
@@ -70,3 +70,13 @@ fi
 ### copy .bazelrc-ci into $HOME/.bazelrc
 ###
 cp "src/dev/ci_setup/.bazelrc-ci" "$HOME/.bazelrc";
+
+###
+### generate .bazelrc-ci.auth
+###
+echo "# Generated by src/dev/ci_setup/setup.sh" > src/dev/ci_setup/.bazelrc-ci.auth
+echo "build --bes_results_url=https://app.buildbuddy.io/invocation/" >> src/dev/ci_setup/.bazelrc-ci.auth
+echo "build --bes_backend=grpcs://cloud.buildbuddy.io" >> src/dev/ci_setup/.bazelrc-ci.auth
+echo "build --remote_cache=grpcs://cloud.buildbuddy.io" >> src/dev/ci_setup/.bazelrc-ci.auth
+echo "build --remote_timeout=3600" >> src/dev/ci_setup/.bazelrc-ci.auth
```

Comment:
I feel like these settings aren't "auth", and should just live in the bazelrc file.

Reply:
Agreed, already moved them.